src/styles/components/tabs.scss

```diff
@@ -4,28 +4,65 @@
 .tabs {
   &__header {
     width: 100%;
-    white-space: nowrap;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
     overflow-x: auto;
-    border-bottom: 0.0625rem solid $colour-platinum;
+    box-shadow: inset 0 -0.1rem 0 0 $colour-platinum;
```

Comment:
It might be good, while touching this file, to replace all SASS colour variables with the franklin Custom Properties that we have defined.